bibtex-actions.el

```diff
@@ -38,6 +38,7 @@
 ;;; Code:
 
 (require 'bibtex-completion)
+(require 'embark)
```

Comment:
Why require embark? I recommend `with-eval-after-load` to register your package with Embark. I think your package can also be used without Embark, so the integration should be optional.